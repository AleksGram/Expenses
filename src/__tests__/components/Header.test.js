import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header';

test('renderHeaderComponet', () => {
    const render = new ReactShallowRenderer();
    render.render(<Header/>);
    expect(render.getRenderOutput()).toMatchSnapshot();
})