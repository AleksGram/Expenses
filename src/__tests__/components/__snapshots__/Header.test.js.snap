// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderPage_test renderHeaderComponet 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    aria-current="page"
    to="/dashboard"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </NavLink>
  <button
    onClick={[MockFunction]}
  >
    Logout
  </button>
</header>
`;
